src/extensions/default/bramble/htmlContent/console.html

```diff
@@ -4,5 +4,9 @@
         <button id="clearConsole" title="{{CONSOLE_CLEAR_TOOLTIP}}" class="clear-button">{{CONSOLE_CLEAR}}</button>
         <button title="{{CONSOLE_CLOSE_TOOLTIP}}"href="#" class="close"></button>
     </div>
-    <div class="console"></div>
+    <div class="console">
+      <div class="helper">
+        <span class="helper-text">{{&CONSOLE_HELPTEXT}}</span>
```

Comment:
From the Mustache docs, this is actually OK to do, and I see it elsewhere in the Brackets code.  You can also do `{{{...}}}` to unescape as well.  So disregard my comment.